shared/ui-components/src/page_components/AppStatusItem.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react"
+import "./AppStatusItem.scss"
+import moment from "moment"
+import { MultiLineAddress } from "../helpers/address"
+import { t } from "../helpers/translator"
+import { Application } from "@bloom-housing/backend-core/client"
+import { LocalizedLink } from "../atoms/LocalizedLink"
+import { Modal } from "../modals/Modal"
+
+// TODO status and lotteryNumber should be loaded from Application
+interface AppStatusItemProps {
+  application: Application
+  status: string
+  setDeletingApplication: (application: Application) => void
+  lotteryNumber?: string
+}
+
+const AppStatusItem = (props: AppStatusItemProps) => {
+  const [showModal, setShowModal] = useState(false)
+  const { application, status, lotteryNumber, setDeletingApplication } = props
+  const listing = application.listing
+  const applicationDueDate = moment(listing.applicationDueDate)
+  const editDate = moment(application.updatedAt)
+  let statusText = t("application.statuses." + status)
+  const inProgress = status == "inProgress"
+  const active = inProgress && moment() < applicationDueDate
+  let statusClassNames
+
+  if (inProgress && moment() > applicationDueDate) {
+    statusClassNames = "is-past-due"
+    statusText = t("application.statuses.neverSubmitted")
+  }
+  if (status == "submitted") statusClassNames = "is-submitted"
+
+  return (
+    <article className="status-item is-editable animated-fade">
+      <div className="status-item__inner">
+        <header className="status-item__header">
+          <h3 className="status-item__title">{listing.name}</h3>
+          <p className="status-item__due">
+            {t("listings.applicationDeadline")}: {applicationDueDate.format("MMMM D, YYYY")}
+          </p>
+        </header>
+
+        <section className="status-item__content">
+          <div className="status-item__details">
+            <p className="status-item__address">
+              <MultiLineAddress address={listing.buildingAddress}></MultiLineAddress>
+            </p>
+            {lotteryNumber && (
+              <p className="status-item__number">
+                {t("application.yourLotteryNumber")} {lotteryNumber}
+              </p>
+            )}
+          </div>
+
+          <div className="status-item__action">
+            <p className="status-item__status">
+              <span className={"status-item__label " + statusClassNames}>
+                {t("application.status")}: {statusText}
+              </span>
+            </p>
+            {status == "submitted" && (
+              <a href="#" className="button small">
```

Comment:
These links don't go anywhere at present?

Reply:
Believe we don't have a application page yet?

Reply:
Ah, yeah I think #472 would need to get merged in first

Reply:
merged!

Reply:
hmmm, I think application view page should work a bit different if we want to use it for showing applications that are already submitted. I think applications should be loaded from API, not via AppSubmissionContext context. What do you think @bk3c @jaredcwhite ? Should I maybe add it to #532 ?

Reply:
I think we should load it from a context, so that we can take advantage of the authentication already being built in with that pattern, but I agree that we should either rename AppSubmissionContext, use ApiClientContext (probably my leading contender), or create a new one.

Adding it to #532 makes sense to me so we can get this merged.